pkg/tkn/tkn.go

```diff
@@ -72,7 +72,7 @@ func AssertClientVersion(binary string) {
 		expectedVersion := os.Getenv("TKN_CLIENT_VERSION")
 		commandResult = cmd.MustSucceed("/tmp/tkn", "version").Stdout()
 		var splittedCommandResult = strings.Split(commandResult, "\n")
-		for i, _ := range splittedCommandResult {
+		for i := range splittedCommandResult {
```

Comment:
Is the `_` causing trouble? afaik it's a common pattern in go

Reply:
pkg/tkn/tkn.go:154:2: S1011: should replace loop with `commandArgs = append(commandArgs, args...)` (gosimple)
	for _, arg := range args {
	^
pkg/tkn/tkn.go:75:10: S1005: unnecessary assignment to the blank identifier (gosimple)
		for i, _ := range splittedCommandResult {

Can we disable this as false positive?